.github/scripts/action.yaml

```diff
@@ -0,0 +1,40 @@
+name: Hermetic library generation
+description: Runs hermetic library generation to produce changed libraries
+inputs:
+  base_ref:
+    description: base branch
+    required: true
+  head_ref:
+    description: head branch
+    required: true
+  image_tag:
+    description: the tag of hermetic build image
+    required: false
+  token:
+    description: PAT
```

Comment:
What is `PAT`?

Reply:
Personal access token.

Reply:
Can we use the full name instead of abbreviations since it's not common? See [style guide](https://developers.google.com/style/abbreviations#creating-abbreviations).

Reply:
Changed to full name.